harness/determined/_generic/_context.py

```diff
@@ -26,15 +26,14 @@ def __init__(
         )
 
     @contextlib.contextmanager
-    def _download_initial_checkpoint(self, checkpoint: Dict) -> Iterator[pathlib.Path]:
+    def _download_initial_checkpoint(self, uuid: str) -> Iterator[pathlib.Path]:
```

Comment:
storage_id?

Reply:
Done.  Also fixed in a variety of other places my code touched.